{{cookiecutter.project_dash_slug}}/Dockerfile

```diff
@@ -30,53 +25,21 @@ COPY --chown=user:user setup.py setup.cfg /srv/
 # Create a dummy service directory for the dependency installation.
 RUN mkdir -p /srv/{{cookiecutter.project_slug}}
 
-# Install the package with pip.
-RUN pip install /srv[prod]
-
 # Copy the entrypoint files for booting the application.
 COPY --chown=user:user docker-entrypoint.sh /srv/
 
-# Set the command executed first when the container run
-ENTRYPOINT ["./docker-entrypoint.sh"]
-
-##
-# dev
-#
-# The dev image installs development-level dependencies in the container and
-# the project to be mounted in the container.
-#
-FROM prod as dev
-
 # Copy required project files into the WORKDIR.
-#COPY --chown=user:user . /srv/
+COPY --chown=user:user . /srv/
 
 # Install the package with pip.
-RUN pip install /srv[dev,test]
+RUN pip install /srv
 
 # Mount a volume at /srv so code can be changed on the fly
 VOLUME /srv
 
-##
-# test
-#
-# The `test` image provides a dedicated container for running the application
-# test suite. It is based on the `src` image so that a developer will have
-# diagnostic tools available in the event of a failure.
-#
-FROM dev as test
-
-# Copy bin scripts locally
-COPY --chown=user:user bin /srv/bin
-
-# Copy test configuration(s)
-COPY --chown=user:user .coveragerc /srv/.coveragerc
-
-# Copy the test suite
-COPY --chown=user:user tests /srv/tests
-
 # Install the test reporter for code climate
 ENV CC_REPORTER_URL=https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
 ADD --chown=user:user "$CC_REPORTER_URL" /srv/cc-test-reporter
 
-# Install the package with pip.
-RUN pip install /srv[test]
+# Set the command executed first when the container run
+ENTRYPOINT ["./docker-entrypoint.sh"]
```

Comment:
I'd prefer to have this on the PATH and not refer to it from a relative path.

Reply:
@kojiromike I will put the full path, but I do not think we should put this on the PATH since we are building an SRV folder for the purpose of knowing where all the stuff goes.